cli/tools/info.rs

```diff
@@ -128,6 +130,98 @@ fn print_cache_info(
   }
 }
 
+fn add_npm_packages_to_json(
+  json: &mut serde_json::Value,
+  npm_resolver: &NpmPackageResolver,
+) {
+  // ideally deno_graph could handle this, but for now we just modify the json here
+  let snapshot = npm_resolver.snapshot();
+  let json = json.as_object_mut().unwrap();
+  let modules = json.get_mut("modules").and_then(|m| m.as_array_mut());
+  if let Some(modules) = modules {
+    if modules.len() == 1
+      && modules[0].get("kind").and_then(|k| k.as_str()) == Some("external")
+    {
+      // If there is only one module and it's "external", then that means
+      // someone provided an npm specifier as a cli argument. In this case,
+      // we want to show which npm package the cli argument resolved to.
```

Comment:
See the output in info/chalk_json.out to see whether this makes sense.

Reply:
IMO this makes sense